src/components/document/annotation-layer.tsx

```diff
@@ -37,11 +39,17 @@ export const AnnotationLayer = observer(function AnnotationLayer({
   const ui = useUIStore();
   const tileApiInterface = useContext(TileApiInterfaceContext);
 
-  const readWriteClass = readOnly ? "read-only" : "read-write";
-  const documentClasses = `.document-content.${readWriteClass} `;
+  // Force rerenders when the layer's size changes
+  const [_resizes, setResizes] = useState(0);
+  useResizeObserver({ref: divRef.current, box: "border-box",
+    onResize() {
+      setResizes(value => value + 1);
+    }
+  });
```

Comment:
1. You can pass in `divRef` (not `divRef.current`) to `useResizeObserver()`.
2. According to the documentation, "By the default the hook will trigger a re-render on all changes to the target element's width and / or height." so you shouldn't need to update your own state value.
```suggestion
  // Force rerenders when the layer's size changes
  useResizeObserver({ref: divRef, box: "border-box"});
```